components/infobox/commons/infobox.lua

```diff
@@ -36,13 +36,20 @@ function Infobox:name(name)
     return self
 end
 
-function Infobox:image(fileName, default)
+function Infobox:image(fileName, default, size)
     if (fileName == nil or fileName == '') and (default == nil or default == '') then
         return self
     end
 
     local infoboxImage = mw.html.create('div'):addClass('infobox-image')
-    local fullFileName = '[[File:' .. (fileName or default) .. '|center|600px]]'
+    size = tonumber(size or '')
+    if size then
+        size = size .. 'px'
+        infoboxImage:addClass('infobox-fixed-size-image')
+    else
+        size = '600px'
+    end
+    local fullFileName = '[[File:' .. (fileName or default) .. '|center|' .. size .. ']]'
```

Comment:
```suggestion
    if size then
        infoboxImage:addClass('infobox-fixed-size-image')
    end
    size = tonumber(size or '600') .. 'px'
    local fullFileName = '[[File:' .. (fileName or default) .. '|center|' .. size .. ']]'
```

Reply:
this will error if someone enters a non-number into size

Reply:
Alright